eZ/Publish/Core/Persistence/Legacy/Content/UrlAlias/SlugConverter.php

```diff
@@ -74,7 +73,6 @@ class SlugConverter
                     'endline_search_normalize',
                     'tab_search_normalize',
                     'specialwords_search_normalize',
-                    'punctuation_normalize',
```

Comment:
Well, as the test shows, "urlalias_compat" is still filtering out commas, without "punctuation_normalize". (FWIW, what "punctuation_normalize" actually does is convert `! ? ,` into periods, which then may or may not be removed by some other filter.)